t/unit/app/test_app.py

```diff
@@ -786,6 +786,17 @@ def my_failover_strategy(it):
         assert self.app.connection('amqp:////value') \
                        .failover_strategy == my_failover_strategy
 
+    def test_amqp_heartbeat_settings(self):
+        # Test default broker_heartbeat value
+        assert self.app.connection('amqp:////value').heartbeat == 0
+
+        # Test passing heartbeat through app configuration
+        self.app.conf.broker_heartbeat = 60
+        assert self.app.connection('amqp:////value').heartbeat == 60
+
+        # Test passing heartbeat as connection argument
+        assert self.app.connection('amqp:////value', heartbeat=30).heartbeat == 30
```

Comment:
flake8 is complaining again because of line length.
Please extract the heartbeat result to a variable.